doc/documentation.rst

```diff
@@ -28,10 +28,10 @@ Documentation of scikit-learn 0.19.dev0
                 <!-- doc versions -->
                     <h2>Other Versions</h2>
                     <ul>
-                        <li>scikit-learn 0.19 (development)</li>
-                        <li><a href="http://scikit-learn.org/stable/documentation.html">scikit-learn 0.18 (stable)</a></li>
-                        <li><a href="http://scikit-learn.org/0.17/documentation.html">scikit-learn 0.17</a></li>
-                        <li><a href="http://scikit-learn.org/0.16/documentation.html">scikit-learn 0.16</a></li>
+                        <script>if (VERSION_SUBDIR != "stable") document.write('<li><a href="http://scikit-learn.org/stable/documentation.html">Stable version</a></li>')</script>
+                        <script>if (VERSION_SUBDIR != "dev") document.write('<li><a href="http://scikit-learn.org/dev/documentation.html">Development version</a></li>')</script>
+                        <li><a href="http://scikit-learn.org/dev/versions.html">Previous versions</a></li>
```

Comment:
This doesn't work for [circle ci preview](https://13946-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/documentation.html). Can't one use relative paths here i.e. `./versions.html`?

Reply:
As below, it can't be a local link if it's to work on /stable and /0.19, etc. But the moment this is merged to master, the link will also work.